lychee-lib/src/helpers/reqwest.rs

```diff
@@ -0,0 +1,46 @@
+use once_cell::sync::Lazy;
+use regex::Regex;
+
+pub(crate) static REQWEST_ERROR_REGEX: Lazy<Regex> =
+    Lazy::new(|| Regex::new(r"error trying to connect:\s?(.*)").unwrap());
```

Comment:
I don't think it's necessary to use regex pattern. This can be done by matching the fixed string.

Reply:
You're right. I'll change it when I get the chance.